### ISSUE TITLE
[Feature] Linking Minecraft and Telegram accounts

### ISSUE THREAD

**Issue by Zakallenko** [kraftwerk28/spigot-tg-bridge#17]
Please, add function for linking TG and MC accounts. So only linked users can send messages via bot in game chat. Also in-game message from TG will contain sender's game nickname instead user's first name.

**Comment by kraftwerk28** [comment]
Sounds like a nice feature, I'll try to implement it

**Comment by kraftwerk28** [comment]
Being late on this as I don't have much free time, but I'll get to it eventually. The only question is: what if the server is `offline: true` and the player changes it's name to someone's else? I know there are plenty of auth plugins (i.e. `AuthMe`), but if neither if installed on the server, then anyone can "steal" the voice and talk anything to tg chat.

**Comment by witheredwojak** [comment]
I don't think that even a slightly public server with online-mode=false will have not an authorization plugin. At least you can give ability to unlink accounts for admins.

**Comment by SabreT00th84** [comment]
This is a good idea but it should be optional
